apidoc2swagger/operationidmapping.txt

```diff
@@ -117,3 +117,30 @@ app/searchext/=searchExt
 /core/deleteversion=deleteVersion
 /core/downloadversionedfile=downloadVersionedFile
 /core/getversions=getVersions
+/core/getauditgeoipdata=getAuditGeoIpData
+/core/cancelsearch=cancelSearch
+/core/clearsavedsearchterms=clearSavedSearchTerms
+/core/dosearch=doSearch
+/core/getwopiediturl=getWOpIEditUrl
+/core/searchtermsearch=searchTermSearch
+/core/addsettofileobject=addSetToFileObject
+/core/getavailablemetadatasets=getAvailableMetaDataSets
+/core/getmetadatasetsforsearch=getMetaDataSetsForSearch
+/core/removesetfromfileobject=removeSetFromFileObject
+/core/saveattributevalues=saveAttributeValues
+/core/acktoasts=ackToasts
+/core/deletetoasts=deleteToasts
+/core/getnotificationstream=getNotificationStream
+/core/gettoasts=getToasts
+/core/getanonymoususerconsent=getAnonymousUserConsent
+/core/getuserconsent=getUserConsent
+/core/privacyconsentaccept=privacyConsentAccept
+/core/getprofileimage=getProfileImage
+/core/getmostactiveshares=getMostActiveShares
+/core/getrecentshareactivityforuser=getRecentShareActivityForUser
+/core/getshareactivityforshare=getShareActivityForShare
+/core/addshortcut=addShortcut
+/core/deleteshortcut=deleteShortcut
+/core/getshortcuts=getShortcuts
+/core/getoourl=getOOUrl
```

Comment:
Is that an o or zero in the name? I would guess that the url contains o and as such we should also use O (capital o) in our method name?

Reply:
These are capital o's